### PR TITLE
Adds validation of Lithuanian license plates.

### PR DESCRIPTION
**Pull request by vanherpen**

### CODE REVIEW

File: src/main/java/net/contargo/types/truck/LithuanianLicensePlateHandler.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package net.contargo.types.truck;
+
+/**
+ * @author  Marius van Herpen - mvanherpen@contargo.net
+ */
+public class LithuanianLicensePlateHandler implements LicensePlateHandler {
```
Comment (geld0r) [inline src/main/java/net/contargo/types/truck/LithuanianLicensePlateHandler.java:21]: Maybe add some examples of typical and supported license plates as well as a link to the documentation used.

File: src/main/java/net/contargo/types/truck/LithuanianLicensePlateHandler.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+     *
+     * <p>A Lithuan license plate consists of a group of 3 letters and 3 digits. There is also the possibility of a
+     * vanity plate, which allow any combination of letters and digits, and has no minimum length. Those will not be
+     * validated.</p>
+     *
+     * @param  value  to be validated, never {@code null}
+     *
+     * @return  {@code true} if the given {@link LicensePlate} is valid, else {@code false}
+     */
+
```
Comment (geld0r) [inline src/main/java/net/contargo/types/truck/LithuanianLicensePlateHandler.java]: empty line?

File: src/test/java/net/contargo/types/truck/LithuanianLicensePlateHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    @Test
+    public void assertIncorrectGroupsInvalid() {
+
+        assertIsNotValid.accept("645 564");
+        assertIsNotValid.accept("645 JKZ");
+        assertIsNotValid.accept("FDS JKR");
+        assertIsNotValid.accept("JKR 65");
+        assertIsNotValid.accept("JKR 6435");
+        assertIsNotValid.accept("JK 645");
```
Comment (geld0r) [inline src/test/java/net/contargo/types/truck/LithuanianLicensePlateHandlerTest.java:50]: Maybe also add a case where no separator at all is used or when too many letters are used in the first block.
Maybe also add a test case for using multiple separators?

**Review by geld0r — CHANGES_REQUESTED** [review]
Looks good, only some small comments regarding documentation and tests.

File: src/test/java/net/contargo/types/truck/LithuanianLicensePlateHandlerTest.java (hunk 3 above)
Comment (vanherpen) [reply src/test/java/net/contargo/types/truck/LithuanianLicensePlateHandlerTest.java:50]: Added some more testcases.

**Pull request merged**